iothub_client/samples/pnp/pnp_simple_thermostat/pnp_simple_thermostat.c

```diff
@@ -122,11 +126,12 @@ static const char g_maxTemperatureSinceRebootFormat[] = "{\"maxTempSinceLastRebo
 // for the solution to correlate the request and its status.
 static const char g_targetTemperatureResponseFormat[] = "{\"targetTemperature\":{\"value\":%.2f,\"ac\":%d,\"av\":%d,\"ad\":\"%s\"}}";
 
-// Response description is an optional, human readable message including more information
-// about the setting of the temperature.  Because we accept all temperature requests, we 
-// always indicate a success.  An actual sensor could optionally return information about
-// a temperature being out of range or a mechanical issue on the device on error cases.
-static const char g_temperaturePropertyResponseDescription[] = "success";
+// Response description is an optional, human readable message including more information about the setting of the temperature.  
+// NOTE: Because these responses are optional, devices with RAM/ROM constraints or running over limited or metered
+// connections should omit these.
+static const char g_temperatureDescriptionSuccess[] = "success";
+static const char g_temperatureDescriptionMisformatted[] = "request misformatted";
```

Comment:
I'm biased to `malformed`.
```suggestion
static const char g_temperatureDescriptionMisformatted[] = "request malformed";
```